src/back-end/lib/resources/affiliation.ts

```diff
@@ -168,22 +185,22 @@ const resource: Resource = {
   delete(connection) {
     return {
       async validateRequestBody(request) {
-        const validatedAffiliationId = await validateAffiliationId(connection, request.params.id);
-        if (isInvalid(validatedAffiliationId)) {
+        const validatedAffiliation = await validateAffiliationId(connection, request.params.id);
+        if (isInvalid(validatedAffiliation)) {
           return invalid({
-            affiliation: getInvalidValue(validatedAffiliationId, undefined)
+            affiliation: getInvalidValue(validatedAffiliation, undefined)
```

Comment:
You can just write `affiliation: validatedAffiliation.value` without a type cast